sklearnex/svm/_common.py

```diff
@@ -98,39 +98,30 @@ def _compute_balanced_class_weight(self, y):
         return recip_freq[le.transform(classes)]
 
     def _fit_proba(self, X, y, sample_weight=None, queue=None):
-        from .._config import config_context, get_config
-
         params = self.get_params()
         params["probability"] = False
         params["decision_function_shape"] = "ovr"
         clf_base = self.__class__(**params)
 
-        # We use stock metaestimators below, so the only way
-        # to pass a queue is using config_context.
-        cfg = get_config()
-        cfg["target_offload"] = queue
-        with config_context(**cfg):
-            try:
```

Comment:
Did you verify that CPU/GPU device is still correctly passed now?

Reply:
I'm pretty sure that before my changes it did not work well because default sklearn config without target_offload has been called here. So, I didn't make things worse. But I agree that it should be investigated too. I will check and create JIRA task if I will find any inconsistencies. I think these changes can be separated to another pull request if they will be necessary.